R/utils-assert-args.R

```diff
@@ -32,3 +32,9 @@ igraph.match.arg <- function(arg, values, error_call = rlang::caller_env()) {
     error_call = error_call
   )
 }
+
+ensure_no_na <- function(x, what, call) {
```

Comment:
It's better to do this in the default and not pass it if not needed in the individual calls. We want `#' @importFrom rlang caller_env` too.

```suggestion
ensure_no_na <- function(x, what, call = caller_env()) {
```